source/features/conversation-filters.tsx

```diff
@@ -12,12 +12,14 @@ function init(): void {
 	commentsLink.lastChild!.textContent = 'Everything commented by you';
 	commentsLink.removeAttribute('target');
 	new SearchQuery(commentsLink).set('is:open commenter:@me');
+	commentsLink.setAttribute('aria-checked', String(commentsLink.href === location.href)); // #4589
```

Comment:
This allow marking the filter as selected, just like the native ones.

![image](https://user-images.githubusercontent.com/46634000/141990900-9df83381-7151-4425-810b-06a4face754b.png)

It follows the behavior of the native filters, that are only shown selected when the current search query matches exactly with the one they represent.